test/unit/Reflection/ReflectionFunctionTest.php

```diff
@@ -76,4 +86,25 @@ public function testStaticCreation()
         $reflection = ReflectionFunction::createFromName('BetterReflectionTest\Fixture\myFunction');
         $this->assertSame('myFunction', $reflection->getShortName());
     }
+
+    public function functionStringRepresentations()
+    {
+        return [
+            ['BetterReflectionTest\Fixture\myFunction', "Function [ <user> function BetterReflectionTest\Fixture\myFunction ] {\n  @@ /home/james/workspace/better-reflection/test/unit/Fixture/Functions.php 5 - 6\n}"],
+            ['BetterReflectionTest\Fixture\myFunctionWithParams', "Function [ <user> function BetterReflectionTest\Fixture\myFunctionWithParams ] {\n  @@ /home/james/workspace/better-reflection/test/unit/Fixture/Functions.php 8 - 9\n\n  - Parameters [2] {\n    Parameter #0 [ <required> \$a ]\n    Parameter #1 [ <required> \$b ]\n  }\n}"],
+        ];
+    }
+
+    /**
+     * @param string $functionName
+     * @param string $expectedStringValue
+     * @dataProvider functionStringRepresentations
+     */
+    public function testStringCast($functionName, $expectedStringValue)
+    {
+        require_once(__DIR__ . '/../Fixture/Functions.php');
+        $functionInfo = ReflectionFunction::createFromName($functionName);
+
+        $this->assertSame($expectedStringValue, (string)$functionInfo);
```

Comment:
These need to use `assertStringMatchesFormat()` and replace hard-coded path - failing tests